packages/components/src/grid/stories/index.tsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+
+/**
+ * Internal dependencies
+ */
+import { View } from '../../view';
+import { Grid } from '..';
+
+const meta: ComponentMeta< typeof Grid > = {
+	component: Grid,
+	title: 'Components (Experimental)/Grid',
+	argTypes: {
+		as: { control: { type: 'text' } },
+		align: { control: { type: 'text' } },
+		children: { control: { type: null } },
+		columnGap: { control: { type: 'text' } },
+		columns: { control: { type: 'number' } },
+		justify: { control: { type: 'text' } },
+		rowGap: { control: { type: 'text' } },
+		rows: { control: { type: 'number' } },
```

Comment:
```suggestion
		rows: {
			table: { type: { summary: 'number' } },
			control: { type: 'number' },
		},
```

If updating `columns` this will need a tweak as well.